src/gui/widgets/InstrumentFunctionViews.cpp

```diff
@@ -137,6 +139,22 @@ InstrumentFunctionArpeggioView::InstrumentFunctionArpeggioView( InstrumentFuncti
 			"number of octaves." ) );
 
 
+	m_arpSkipKnob->setLabel( tr( "SKIP" ) );
+	m_arpSkipKnob->setHintText( tr( "Skip rate:" ) + " ", " " + tr( "%" ) );
```

Comment:
The description is trimmed, so there is no need for a space after `tr( "Skip rate:" )`.